spark/src/test/scala/com/gu/tableversions/spark/VersionedDatasetSpec.scala

```diff
@@ -130,8 +150,64 @@ class VersionedDatasetSpec extends FlatSpec with Matchers with SparkHiveSuite {
     tableUpdate.userId shouldBe userId
   }
 
+  "Inserting multiple records into the same partition" should "write the correct data to the filesystem" in {
+    spark.sparkContext.hadoopConfiguration.set("fs.versioned.baseFS", "file")
```

Comment:
Could replace these calls with a single one outside of each test? (Perhaps replacing it with a method call if we add one for this...)